src/zjs_ocf_common.c

```diff
@@ -223,7 +223,7 @@ void zjs_set_uuid(char *uuid)
 
 static void platform_init(void *data)
 {
-    u32_t size;
+    jerry_value_t size;
```

Comment:
This is wrong too, you can't later assign an int to size like that. jerry_value_t is not a number, it's encoded - the bottom bits specify type.